pandas/tests/reshape/test_tile.py

```diff
@@ -488,6 +513,47 @@ def test_datetime_cut(self):
         result, bins = cut(data, 3, retbins=True)
         tm.assert_series_equal(Series(result), expected)
 
+    @pytest.mark.parametrize('bins', [
+        3, [Timestamp('2013-01-01 04:57:07.200000'),
+            Timestamp('2013-01-01 21:00:00'),
+            Timestamp('2013-01-02 13:00:00'),
+            Timestamp('2013-01-03 05:00:00')]])
```

Comment:
@mroeschke trying to address #47772 im getting a failure in this test bc i think the bins here need to be tzaware for this to make sense.  without them being tzaware i think this should raise.  but if i pass tz="US/Eastern" to the Timestamp constructor then i just get a regular failure.  pls advise

Reply:
Looks like if i do `Timestamp(..., tz="UTC").tz_convert("US/Eastern")` in the bins the the rest of the test passes.  is that what you would expect?

Reply:
I would have expected just passing tz="US/Eastern" into `Timestamp` to also pass this test and not needing to localize to UTC first. Is the failure with all `box` types?

Reply:
> Is the failure with all box types?

Yes